tests/unit/DebugUtilsTest.ts

```diff
@@ -0,0 +1,546 @@
+import DebugUtils from '@libs/DebugUtils';
+import CONST from '@src/CONST';
+import type {Report, ReportAction} from '@src/types/onyx';
+
+const MOCK_REPORT: Report = {
```

Comment:
Is possible to use [createRandomReport()](https://github.com/Expensify/App/blob/main/tests/utils/collections/reports.ts) here?